cmake/toolchain.linux-x64.cmake

```diff
@@ -0,0 +1,17 @@
+# Name of the target platform
+set(CMAKE_SYSTEM_NAME Linux)
+
+# Version of the system
+set(CMAKE_SYSTEM_VERSION 1)
+
+# specify the cross compiler
+set(CMAKE_C_COMPILER gcc)
+set(CMAKE_SHARED_LINKER_FLAGS "-Wl,--no-undefined" CACHE STRING "" FORCE)
+set(CMAKE_EXE_LINKER_FLAGS "-Wl,--no-undefined" CACHE STRING "" FORCE)
+
+# compiler flags
+set(CMAKE_C_FLAGS "--std=gnu99 -Werror=implicit-function-declaration" CACHE STRING "" FORCE)
+
```

Comment:
Those --std options may be set with the CMake variables CMAKE_C_STANDARD and CMAKE_CXX_STANDARD_REQUIRED in the main CMake files.

btw: I see no sense in providing toolchain files at all, those are highly user and toolchain dependend. Cross build systems come with their own toolchain files anyway.